install-deps.sh

```diff
@@ -29,9 +29,16 @@ function debs() {
             update-alternatives --install /usr/bin/gcc gcc-8 /usr/bin/gcc-8  100
         fi
     fi
-
+    if [[ ${UBUNTU_CODENAME} == "xenial" ]]; then
+        cmake_version="3.14.0-rc2"
```

Comment:
@noahdesu  - this is the issue w/ the current docker build.

Reply:
what issue? i haven't seen any xenial failures.

Reply:
So i had to upgrade cmake to 3.7 because I wanted to simplify the build via `SOUCE_SUBDIR`. so on xenial we just install cmake.